ntp/ntp.cabal

```diff
@@ -40,6 +35,28 @@ Library
                         OverloadedStrings
                         MonadFailDesugaring
 
+Executable ntp-app
```

Comment:
I presume this is for experimenting during development and not something that's permanent.

Reply:
or `ntp-client-demo`